fetch.go

```diff
@@ -26,29 +27,63 @@ type PullRequest struct {
 	Url string
 }
 
-func main() {
-	resp, err := http.Get(IssueUrl)
+type LinkHeader struct {
```

Comment:
Is this `struct` used anywhere? Side note: I was thinking about creating a `struct` as well, but then I decided to use a `map` because of the... mapping :smile:, so it's easy to extract the `next` and `last` keys that the GitHub API puts in the headers.

Now that I'm thinking, we can have a `struct` that is something like:
```go
type LinkHeader struct {
    Next string
    Last string
}
```

What do you think?

Reply:
Well, practically speaking, I'm wondering if it makes sense to return `Last`. At the end what we care about is to go to the next page, not to the last one.

Reply:
On page=2, the header will also return "first" as a value.

Staying with map[string]string is fine for now.